README.md

```diff
@@ -141,6 +141,26 @@ and pierre.casteran@labri.fr.
   patches from Coq developers when they introduce a breaking change (this is
   particularly recommended for plugins).
 
+- **What license to choose for a coq-community project?**
+
+  The only strict requirement is to use an open source license.
```

Comment:
What do we mean by open source licenses? I think we should either link to SPDX's list or to the OSI list. I would go with the either only the former or both.

Reply:
SPDX contains more than open source license. Yes, open source is per OSI definition. I'll add a link (https://opensource.org/licenses).